src/com/synopsys/integration/pipeline/jenkins/JenkinsScriptWrapperImpl.groovy

```diff
@@ -84,6 +87,33 @@ class JenkinsScriptWrapperImpl implements JenkinsScriptWrapper {
         return bat(command, returnStdout)
     }
 
+    @Override
+    void executeCommandWithHttpStatusCheck(String command, String expectedHttpStatusCode, String jsonResponseFileName, String githubCredentialsId, PipelineConfiguration pipelineConfiguration) {
```

Comment:
Talked about this with Paulo: Can you convert receivedHttpStatusCode to an int, have expectedHttpStatusCode be an int, and then compare int->int?

Reply:
Yes, I have updated these to ints by changing the type of expectedHttpStatusCode and using Integer.parseInt() for receivedHttpStatusCode